src/core/utilities.cpp

```diff
@@ -769,6 +769,15 @@ QString ScrubUrlQueries(const QString& str) {
   return QString(str).replace(rx, "\\1 (query removed)");
 }
 
+QString MakeBugReportUrl(const QString& title) {
+  // Example:
+  // https://github.com/clementine-player/Clementine/issues/new?title=New%20bug
+  return QString(
+             "https://github.com/clementine-player/Clementine/"
+             "issues/new?title=") +
+         QUrl::toPercentEncoding(title);
```

Comment:
Can we use the `QUrl` helpers for query items here instead?

Reply:
Done